src/PrettierESLintLinter.php

```diff
@@ -128,6 +134,10 @@ protected function parseLinterOutput($path, $err, $stdout, $stderr) {
       return false;
     }
 
+    if($this->getData($path) == $stdout){
```

Comment:
Spaces before and after the outer parens to match the surrounding style, please.